strawberry_django/utils.py

```diff
@@ -85,15 +85,29 @@ def is_similar_django_type(a, b):
     return True
 
 
+# dirty workaround, but similar to the one of dataclasses
+# it would be better to use dataclasses.fields
+def _is_not_classvar(annotation, namespace):
+    if isinstance(annotation, str):
+        if annotation.startswith("ClassVar["):
+            annotation = namespace["ClassVar"]
+        elif annotation.startswith("typing.ClassVar["):
+            annotation = namespace["typing"].ClassVar
```

Comment:
Although this solution works in general cases, it has some issues:

1) `ClassVar` can be imported from other modules like `typing_extensions`, the `typing` module can be aliased to something else, or the user can have a "compat" module from typing to import it from there

2) The `ClassVar` can be inside a `if TYPE_CHECKING`, specially when using future annotations, and thus would not be on namespace

Having said that, this lib already has a lot of those issues, specially with future annotations, based on the fact that it tries to parse annotations when creating the class, instead of delaying to when the type is requested from the field.

I refactored the `type` funtion on strawberry django plus in this PR (https://github.com/blb-ventures/strawberry-django-plus/pull/235) to solve those issues that I mentioned, and I'll soon (probably this weekend) port that refactoring to this lib as well, meaning that this annotations code will get thrown away.

So, we can either merge this even with those issues based on the fact that the facky solution will have a "short life", or we can avoid it based on the fact that a better overall solution will be made really soon.

What do you think about that? Also, any suggestions @patrick91 ?

Reply:
let's do the solution you think it is best @bellini666 😊

and thanks @devkral for all your PRs and work! <3

Reply:
> let's do the solution you think it is best @bellini666 blush
> 
> and thanks @devkral for all your PRs and work! <3

Ok, lets merge this then! :)